package/firmware/layerscape/ls-rcw/patches/0001-rcw-support-byte-swapping-without-tclsh-tool.patch

```diff
@@ -32,7 +32,7 @@ index 0000000..386310e
 --- /dev/null
 +++ b/byte_swap.py
 @@ -0,0 +1,32 @@
-+#!/usr/bin/env python
++#!/usr/bin/env python2
```

Comment:
It doesn't make sense to me, as SetupHostCommand is going to create proper python symlink in the staging/host/bin directory which is then first in the path, so it should work as it is.

Reply:
I need to have a second look, maybe SetupHostCommand is broken in some cases then. I started because build broke for me on Fedora 31

Reply:
The issue is that if `./staging_dir/host/bin/python` already exists, SetupHostCommand doesn't seem to update it when it rerun